### ISSUE TITLE
Dataform cli runs all commands twice

### ISSUE THREAD

**Issue by mantmic** [dataform-co/dataform#754]
I've noticed that when running the dataform cli commands are run twice. 
This can cause issues when dataform creates a table. Dataform appears to attempt to run the create table statement twice, causing the second command to error. Notice how dataform printed the 'Compiling...' and 'Running' messages twice. 

```
Compiling...

Compiling...

Compiled successfully.

Tue, 19 May 2020 03:18:20 GMT tedious deprecated The default value for `config.options.enableArithAbort` will change from `false` to `true` in the next major version of `tedious`. Set the value to `true` or `false` explicitly to silence this message. at ../../../../../opt/hostedtoolcache/node/12.10.0/x64/lib/node_modules/@dataform/cli/node_modules/mssql/lib/tedious/connection-pool.js:61:23
Compiled successfully.

Running...

Running...

Dataset creation disabled:  ci_test.aurion_t006_emergency_contacts [table]
Dataform encountered an error: There is already an object named 'ci_assertions_test' in the database.

RequestError: There is already an object named 'ci_assertions_test' in the database.
CREATE SCHEMA failed due to previous errors.
    at handleError (/opt/hostedtoolcache/node/12.10.0/x64/lib/node_modules/@dataform/cli/node_modules/mssql/lib/tedious/request.js:366:15)
    at Connection.emit (events.js:209:13)
    at Parser.<anonymous> (/opt/hostedtoolcache/node/12.10.0/x64/lib/node_modules/@dataform/cli/node_modules/tedious/lib/connection.js:832:12)
    at Parser.emit (events.js:209:13)
    at Parser.<anonymous> (/opt/hostedtoolcache/node/12.10.0/x64/lib/node_modules/@dataform/cli/node_modules/tedious/lib/token/token-stream-parser.js:37:14)
    at Parser.emit (events.js:209:13)
    at addChunk (/opt/hostedtoolcache/node/12.10.0/x64/lib/node_modules/@dataform/cli/node_modules/tedious/node_modules/readable-stream/lib/_stream_readable.js:298:12)
    at readableAddChunk (/opt/hostedtoolcache/node/12.10.0/x64/lib/node_modules/@dataform/cli/node_modules/tedious/node_modules/readable-stream/lib/_stream_readable.js:280:11)
    at Parser.Readable.push (/opt/hostedtoolcache/node/12.10.0/x64/lib/node_modules/@dataform/cli/node_modules/tedious/node_modules/readable-stream/lib/_stream_readable.js:241:10)
    at Parser.Transform.push (/opt/hostedtoolcache/node/12.10.0/x64/lib/node_modules/@dataform/cli/node_modules/tedious/node_modules/readable-stream/lib/_stream_transform.js:139:32)

##[error]Bash exited with code '1'.

```

In the cli index.js file we have lines 23404 to 23408 

```
if (require.main === module) {
    runCli();
}

runCli();
```

I believe when that if statement evaluates to true, runCli() is executed twice causing all commands to be run twice. When I comment out the final runCli() command all my dataform commands run as expected.

```
if (require.main === module) {
    runCli();
}

//runCli();
```

Perhaps this is a funky issue with the build process? 

I'm not too familiar with typescript, but I wonder if removing line 2 from packages/@dataform/cli/index.ts would solve this issue.

```
import { runCli } from "df/cli";
runCli();

```

**Comment by BenBirt** [comment]
Thanks for this report! Will fix ASAP

**Comment by BenBirt** [comment]
This should now be fixed in `1.6.8`!
